### PR TITLE
Update bootstrap.loader.js

### LINKED ISSUE #129: Unwanted debug logs because process.env.DEBUG is set by npm module "debug"
Hello

I'm experiencing an issue with [global.__DEBUG__ = loglevel === 'debug' || process.env.DEBUG;](https://github.com/shakacode/bootstrap-loader/blob/master/src/bootstrap.loader.js#L70).

Indeed, I'm using [debug](https://github.com/visionmedia/debug) to log information during my gulp/webpack process. With this framework, I can do:

    import Debug from 'debug';
    const debug = Debug('evenium:js-build-toolsbuild-web');
    ....
    debug('Hello');
    ....

In command line, I can do `set DEBUG=evenium:*` before calling my gulp process. As a result, got at screen all logs called with a debug function instanciated using the string "evenium:":

image

My probleme is that I got a lot of logs from bootstrap loader even if I have set in bootstraprc `loglevel: disabled`.

I understand that  [global.__DEBUG__ = loglevel === 'debug' || process.env.DEBUG;](https://github.com/shakacode/bootstrap-loader/blob/master/src/bootstrap.loader.js#L70). activates the bootstrap debug logs because `process.env.DEBUG` is set.

I should be able to defined `DEBUG` for my own need (aka using [debug](https://github.com/visionmedia/debug)) without having bootstrap logs on screen.

### PR DESCRIPTION
**Pull request by damienleroux**
process.env.DEBUG must target a specific value to avoid conflit with other possible values (DEBUG namespaces)

Resolves #129

---
This change is [Reviewable](https://reviewable.io/reviews/shakacode/bootstrap-loader/130)

### CODE REVIEW

File: src/bootstrap.loader.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ module.exports.pitch = function(source) {
 
   const loglevel = config.loglevel;
 
-  global.__DEBUG__ = loglevel === 'debug' || process.env.DEBUG;
+  global.__DEBUG__ = loglevel === 'debug' || process.env.DEBUG === true || process.env.DEBUG === "true" || process.env.DEBUG === "yes" || process.env.DEBUG === 1;
```
Comment (justin808) [inline src/bootstrap.loader.js]: I'd like to see this logic in a function, and downcase before checking, so TRUE passes

**Comment by justin808** [comment]
some feedback

also, be sure to lint

`npm run lint`


the long line looks too long

---

Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions.

---

*[src/bootstrap.loader.js, line 70 \[r2\]](https://reviewable.io:443/reviews/shakacode/bootstrap-loader/130#-KPgZong4SziyjOL19Sz:-KPgZong4SziyjOL19T-:b-idqjao) ([raw file](https://github.com/shakacode/bootstrap-loader/blob/a821a091cc39c90ee758c0ad2063af7160dea147/src/bootstrap.loader.js#L70)):*
> ```JavaScript
>   function isDebugEnable(){
>     const loglevel = config.loglevel;
>     const globalDebugVar = process.env.DEBUG;
> ```

let's lowercase here 

```js
const globalDebugVar = process.env.DEBUG && process.env.DEBUG.toString().toLowerCase();
```

Please confirm this works.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/shakacode/bootstrap-loader/130#-:-KPg_BJTmr8sFNcb4VMr:br64773)*

**Comment by damienleroux** [comment]
Sorry for that. I had edited it from github..

Damien leroux

**Comment by damienleroux** [comment]
Hello! I had access to my computer at least! I fix lint errors. Thank you

**Comment by justin808** [comment]
@alexkval Please take a look. Looks OK to me if it passes linting.

Do we release for both 1.x and 2.x, @damienleroux ?

---

Reviewed 1 of 1 files at r2, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[src/bootstrap.loader.js, line 70 \[r1\]](https://reviewable.io:443/reviews/shakacode/bootstrap-loader/130#-KPftZeTS5BHtRdjPfOZ-r1-70:-KPqPJnByPfhpywPkCvc:b-5nwxqp) ([raw file](https://github.com/shakacode/bootstrap-loader/blob/c9d54a778fd9db0d21a0699be4ddac5745628e38/src/bootstrap.loader.js#L70)):*
<details><summary><i>Previously, justin808 (Justin Gordon) wrote…</i></summary>
> I'd like to see this logic in a function, and downcase before checking, so TRUE passes
</details>
I'm surprised that linting passed the fall through case statement.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/shakacode/bootstrap-loader/130#-:-KPqPR8z_DC52U59Tqlf:b54qpcz)*

**Comment by damienleroux** [comment]
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[src/bootstrap.loader.js, line 70 \[r1\]](https://reviewable.io:443/reviews/shakacode/bootstrap-loader/130#-KPftZeTS5BHtRdjPfOZ-r1-70:-KPqWJXqU2RC0p2YmoEW:b-du6d41) ([raw file](https://github.com/shakacode/bootstrap-loader/blob/c9d54a778fd9db0d21a0699be4ddac5745628e38/src/bootstrap.loader.js#L70)):*
> I'm surprised that linting passed the fall t
hum, why not?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/shakacode/bootstrap-loader/130)*

**Comment by damienleroux** [comment]
Sure. if it's ok for you I can make a pull request on V1 with the required code?

---

Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/shakacode/bootstrap-loader/130#-:-KPqXfoR0gshMdR9wwtf:b8960kq)*

**Comment by justin808** [comment]
@damienleroux that would be appreciated, both on the v1 branch and master!

**Comment by AlexKVal** [comment]
Hi @damienleroux
It seems it is ready to be merged.
The last stroke: I would rename `isDebugEnable` => `isDebugEnabled`

--
I'll port this PR to `v1` myself

**Comment by AlexKVal** [comment]
@justin808 I could (squash, rename and) merge this as is and then make this trivial renaming `isDebugEnable` => `isDebugEnabled` myself with one more straight commit to master but I'm not sure about such "moves" yet. 
So I decided to wait for your decision or damienleroux fixing himself.

**Comment by damienleroux** [comment]
Thank you for handling v1. On master I renamed it  `isDebugEnabled` :)

**Comment by AlexKVal** [comment]
Thank you @damienleroux for the contribution !
🍒

**Comment by AlexKVal** [comment]
As it turned out it still contains bugs and doesn't work as expected.
I'm working on fixes here https://github.com/shakacode/bootstrap-loader/pull/133.

**Pull request merged**